### ISSUE TITLE
Test Program gain

### ISSUE THREAD

**Issue by DuncanBennett2** [BostinTechnology/RadarPi#35]
There is no option in the test program to set the gain

**Comment by mattb1969** [comment]
That is correct, there is no specific option to control the gain from within it.
However, the gain is set automatically by the test routines. There are 4 gain settings that are cycled through, gain values of 1, 10, 40 and 157 used in the following routines
- Range Test
- Display Readings Test

I think these routines provide a test for gain? What other functionality would you like?
